src/App.vue

```diff
@@ -13,33 +13,37 @@
 </template>
 
 <script>
-import { watch } from 'vue';
-import { mapActions } from 'vuex';
+import { onMounted, watch, computed } from 'vue';
+import { useStore } from 'vuex';
 import { useModal } from '@/composables/useModal';
+import { useI18n } from '@/composables/useI18n';
 
 export default {
   setup() {
+    const store = useStore();
     const { modalOpen } = useModal();
+    const { loadLocale } = useI18n();
+
+    onMounted(() => {
+      store.dispatch('init');
+      loadLocale();
```

Comment:
Shouldn't `loadLocale()` be before the init with an `await`? We don't want that the UI appear in English then switch to another language.

Reply:
I tested that, but it's probably a good safety measure anyway.